cfpb/consumerfinance.gov
# Add expandable modules to sublanding page type

We don't currently make the expandable modules available on the sublanding page type, but we've found a use case for them in our redesign of the Submit a Complaint page ([previewable on Beta](https://beta.cfpb.gov/complaint-compact-reversed-3/)). 

We plan to apply the FAQPage schema to the expandable content on this page but will add that functionality in a future PR.


## Additions

- Expandable modules on sublanding page type


## How to test this PR

1. Check out branch and run migrations. 
2.  Create a sublanding page and verify that you can add an expandable group


## Checklist


  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
